### DISCUSSION TITLE
Does ImageMagick support DX10 in DDS?

### DISCUSSION THREAD

**Discussion by codecat** [ImageMagick/ImageMagick#2439] [Help]
I've been trying to look for a cross-platform way to convert DDS textures that use the DX10 compression/encoding. ImageMagick seems to throw errors at me when I try to convert it:

```
» magick convert 5ef7825d6a479.dds test.png
convert: image type not supported `/Users/nimble/Downloads/5ef7825d6a479.dds' @ error/dds.c/ReadDDSImage/1767.
```

I think `texconv` (which is part of [DirectXTex](https://github.com/Microsoft/DirectXTex)) can do it, but that seems to only be available on Windows.

My Mac can display the dds texture in Preview, so I don't think DX10 is something specific to Windows either, so it should be able to decompress the texture cross-platform.

[Link to the relevant file](http://trackmania-prod-nls-file-store-s3.mdc.akamaized.net/club/25/screen_2x1/5ef7825d6a479.dds).

**Comment by fmw42** [comment]
What is your IM version?  Try

`magick DDS:5ef7825d6a479.dds test.png`

Does that work? If not, can you post an example input image?

Note that in ImageMagick 7, use magick, not magick convert. For all other tools preface with magick, such as magick identify.

**Reply by codecat** [reply]
I'm using `7.0.10-28 Q16 x86_64 2020-08-16`. 

I get the same error with the `DDS:` prefix.

```
» magick DDS:5ef7825d6a479.dds test.png
magick: image type not supported `5ef7825d6a479.dds' @ error/dds.c/ReadDDSImage/1767.
```

Here's also a [link to the relevant file](http://trackmania-prod-nls-file-store-s3.mdc.akamaized.net/club/25/screen_2x1/5ef7825d6a479.dds).

**Reply by fmw42** [reply]
I can confirm the same issue on my Mac OSX Sierra with IM 7.0.10.28 Q16. A developer will need to look at the file.
